zerver/views/streams.py

```diff
@@ -629,7 +630,9 @@ def add_subscriptions_backend(
     # Validation of the streams arguments, including enforcement of
     # can_create_streams policy and check_stream_name policy is inside
     # list_to_streams.
-    existing_streams, created_streams = list_to_streams(stream_dicts, user_profile, autocreate=True)
+    existing_streams, created_streams = list_to_streams(
+        stream_dicts, user_profile, autocreate=True, is_default_stream=is_default_stream
```

Comment:
I am not sure what is the correct way to do this. For all other properties we pass them in `stream_dicts`, but `is_default_stream` is different as it is not a field in `Stream` object. We can include it in `stream_dicts` for consistency though. Will leave this as it is for now and can take feedback from Tim when he reviews.